packages/create-instantsearch-app/src/cli/index.js

```diff
@@ -201,6 +201,89 @@ const getQuestions = ({ appName }) => ({
       when: ({ appId, apiKey, indexName }) =>
         attributesForFaceting.length === 0 && appId && apiKey && indexName,
     },
+    {
+      type: 'confirm',
+      name: 'useAutocomplete',
```

Comment:
I feel like some of the other code would be easier to read if this key was:
```suggestion
      name: 'autocomplete',
```

Reply:
Addressed in [5b887e3](https://github.com/algolia/instantsearch/pull/5857/commits/5b887e37655cee0acc24630e81f331c3a4f54c7a).